File: selectionsort.cpp
#include<iostream>
#define n 5
using namespace std;

int a[n];
int i;
int j;
int temp;

void displayy()
{
    for(i=0;i<n;i++)
    {
        printf("%d ",a[i]);
    }
}
void selectionsort()
{
    for(i=0;i<n-1;i++)
    {
        for(j=i+1;j<n;j++)
        {
            if(a[j]<a[i])
            {
                temp=a[j];
                a[j]=a[i];
                a[i]=temp;
            }


            
        }
    }


}

int main()
{
    for(i=0;i<n;i++)
    {
        scanf("%d ",&a[i]);
    }


    displayy();

    printf("running selection sort\n");

    selectionsort();

    displayy();



    return 0;


} 

File: searching&sorting/selectionsort.js
class Solution
{
  //Function to sort the array using selection sort algorithm.
  selectionSort(arr,n){
    var i;
    var j;
    var temp;
    
    for(i=0;i<n-1;i++){
        for(j=i+1;j<n;j++){
            if(arr[i]>arr[j]){
                temp=arr[j];
                arr[j]=arr[i];
                arr[i]=temp;
            }
        }
    }
    return arr;
  }
    
}
